tests/unit/models/reddit/test_subreddit.py

```diff
@@ -61,6 +64,32 @@ def test_hash(self):
         assert hash(subreddit2) != hash(subreddit3)
         assert hash(subreddit1) != hash(subreddit3)
 
+    @mock.patch(
+        "praw.Reddit.post",
+        return_value={"json": {"data": {"websocket_url": ""}}},
+    )
+    @mock.patch("praw.models.Subreddit._upload_media", return_value="")
+    @mock.patch("websocket.create_connection")
+    def test_invalid_media(self, connection_mock, __, _):
+        connection_mock().recv.return_value = json.dumps(
+            {"payload": {}, "type": "failed"}
+        )
+        with pytest.raises(MediaPostFailed):
+            self.reddit.subreddit("test").submit_image(
+                "Test",
+                os.path.abspath(
+                    os.path.join(
+                        __file__,
+                        "..",
+                        "..",
+                        "..",
+                        "..",
+                        "integration",
+                        "files," "test.jpg",
```

Comment:
I think the comma is in the wrong place with respect to `files` here. It makes me wonder how this test is even working as it should if the file isn't opening properly... maybe the path is irrelevant to the test -- if so, consider simply making it `dummy_path`.

Reply:
I don't think the path of the test actually matters, since the `_upload_media` class is mocked.

Reply:
Right, so in that case consider simplifying with `dummy_path`.